portal/src/pages/search.vue

```diff
@@ -188,17 +191,15 @@ export default {
       }
       this.executeSearch(changed)
     },
-    loadMore() {
+    onLoadPage(page) {
       const { search, materials } = this
       if (materials && search) {
-        const { page_size, page, records_total } = materials
-
-        if (records_total > page_size * page) {
-          this.$store.dispatch('searchNextPageMaterials', {
-            ...search,
-            page: page + 1,
-          })
-        }
+        search.page = page;
+        this.$router.push(
+          generateSearchMaterialsQuery(this.search, this.$route.name)
+        )
+        this.$store.dispatch('searchMaterials', search)
```

Comment:
Als er al een push wordt gedaan op de router, is de dispatch dan nog noodzakelijk? Verdwijnt de result daarvan dan niet gewoon als de page herlaadt?

Reply:
ik denk t idd, zal t ff checken

Reply:
nee, de router push zorgt ervoor dat de url word aangepast, maar er vind geen reload plaats , hiervoor kan een         this.$router.go(); gedaan worden, maar dat herlaad de volledige pagina